charts/chatwoot/templates/_helpers.tpl

```diff
@@ -193,10 +193,10 @@ Set redis URL
 */}}
 {{- define "chatwoot.redis.url" -}}
 {{- if .Values.redis.enabled -}}
-    "redis://:{{ .Values.redis.auth.password }}@{{ template "chatwoot.redis.host" . }}:{{ template "chatwoot.redis.port" . }}"
+    redis://:{{ .Values.redis.auth.password }}@{{ template "chatwoot.redis.host" . }}:{{ template "chatwoot.redis.port" . }}
 {{- else if .Values.env.REDIS_TLS -}}
-    "rediss://:{{ .Values.redis.password }}@{{ .Values.redis.host }}:{{ .Values.redis.port }}"
+    rediss://:$(REDIS_PASSWORD)@{{ .Values.redis.host }}:{{ .Values.redis.port }}
```

Comment:
`$(REDIS_PASSWORD)` this variable is not being substituted by `helm` templating engine.

Reply:
The value shouldn't be substituted by Helm. Rather, when the container is started it's substituted by Kubernetes from the environment variable mounted from the secret.

Reply:
@DavidSpek Gotcha. Thank you for confirming.